Documentation/networking/sch_skbprio.txt

```diff
@@ -0,0 +1,24 @@
+	  SKB Priority Queue
+ 	=================================
+ 
+   This qdisc schedules a packet according to skb->priority, where a higher
+   value places the packet closer to the exit of the queue. When the queue is
+   full, the lowest priority packet in the queue is dropped to make room for
+   the packet to be added if it has higher priority. If the packet to be added
+   has lower priority than all packets in the queue, it is dropped.
```

Comment:
Take off the indentation of the whole text (not only this paragraph) since this is no longer code, but documentation.